src/App.js

```diff
@@ -1,29 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Impressive job @Trust-007, One thing you forget here is to make the `App` component a `class-based` component. This is to make sure you understand building a react app with class-based components.